bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -202,6 +218,18 @@
 		<description>DC voltage</description>
 		<state pattern="%.1f V" readOnly="true"></state>
 	</channel-type>
+	<channel-type id="udc2">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>DC Voltage 2</label>
+		<description>DC voltage</description>
+		<state pattern="%.1f V" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="udc3">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>DC Voltage</label>
+		<description>DC voltage 3</description>
+		<state pattern="%.1f V" readOnly="true"></state>
+	</channel-type>
```

Comment:
i18n properties file needs to be regenerated.

Reply:
Thanks, I forgot. I've regenerated the i18n file now.